torch/csrc/jit/constants.cpp

```diff
@@ -88,6 +88,14 @@ RegisterOperators reg({
             push(stack, s);
             return 0;
           };
+        } else if (type == WorldType::get()) {
```

Comment:
Hmm we should never have World constants. There really shouldn't be *any* way to create values of this type. It should be taken as an input to every graph, threaded through all ops we want to serialize, and returned. At the end we should have a pass that removes those (it's only a few lines of code - traverse the path of its uses and remove all of them).

Reply:
discussed below